data/property-ops-dictionary--postgres.json

```diff
@@ -2,322 +2,464 @@
   "boolean": [
     {
       "op": "exists",
-      "description": "exists with any value"
+      "description": "exists with any value",
+      "example": "{\n  \"propertyId\": \"subscribed\", \n  \"operation\": {\"op\": \"exists\" }\n}",
+      "caption": "All profiles that do not have data about subscription status"
```

Comment:
This file is actually generated from core, via https://github.com/grouparoo/grouparoo/blob/main/core/bin/build_static_refs.  The goal was to allow core to be the keeper of these rules, and then to have a way to extract and copy them in to this site.  However, we've learned that we need this extra information (example, caption) that core doesn't have. 

Does that mean that:
A. The example & caption should be moved into core so we can re-use it elsewhere
B. There's sufficiently extra/different information that the www site needs that core doesn't have so we should no longer build_static_refs

cc @seancdavis - what do you suggest?

Reply:
Two thoughts:

First, I lean toward B. I don't think core should have anything it's not using, and I think it's totally fine for www to build atop what it can bring from core. That said, because we're adding more, automating this process then feels a little goofy, since it'd take manual intervention from www anyways.

That's why I end up on B. Though it's a little painful that it feels like we're going more manual. But at this stage of the game, that feels fine.

Second, the structure of `example` in this object is painful. It looks tedious to put together and prone to errors. I'd suggest making it an object, then stringifying it and displaying how you'd like on the front end.

Reply:
@evantahler per @seancdavis ' comment, B sounds like a logical route for me to go unless you have any other thoughts?

Reply:
👍 for "B" from me! 🐝

Reply:
@seancdavis  doing the stringified JSON tidied things up quite a bit, thanks!